account.go

```diff
@@ -0,0 +1,180 @@
+package client
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+// Get information about your account. See https://api.omg.lol/#token-get-account-retrieve-account-information
+func (c *Client) GetAccountInfo() (*Account, error) {
+	req, err := http.NewRequest("GET", fmt.Sprintf("%s/account/%s/info", c.HostURL, c.Auth.Email), nil)
```

Comment:
All the methods are consts in the `http` library: https://pkg.go.dev/net/http#pkg-constants